ca_gazebo/launch/create_empty_world.launch

```diff
@@ -4,6 +4,13 @@
   	<arg name="paused" value="$(optenv PAUSED false)"/>
 	<arg name="gui" value="$(optenv GUI true)"/>
 	<arg name="debug" value="$(optenv DEBUG false)"/>
+	<param name="create1/amcl/initial_pose_x" value="-3"/>
```

Comment:
There is one issue here:

When the `env` is not `empty`, these parameters will override the ones from the other launch files. For that, we should include them only if env == empty. A way of doing this is including these parameters in another launch file. Do you have a better idea?

Reply:
I guess i found a workaround, maybe you can check it? It's working now